server/pulp/server/async/scheduler.py

```diff
@@ -317,15 +327,50 @@ def tick(self):
         :return:    number of seconds before the next tick should run
         :rtype:     float
         """
-        ret = super(Scheduler, self).tick()
         self._failure_watcher.trim()
 
+        # Setting the celerybeat name
+        celerybeat_name = SCHEDULER_WORKER_NAME + "@" + str(celery_name) + "@" + platform.node()
```

Comment:
This should be changed back to `SCHEDULER_WORKER_NAME + "@" + platform.node()`.

The normal init scripts will only allow 1 scheduler per hostname and so we no longer need this. Also with this implementation every time `pulp_celerybeat` is restarted the new one can't recognize that it already had the lock because the name has changed with each restart.